### PR TITLE
ToggleTab added, switch player, get player score and new game functions working

### PR DESCRIPTION
**Pull request by Grub2801**
Bugs:
when I update Player 2 score on the scoreboard, it also updates player 1's score with the same value (line 105)
new game function does not clear scoreboard of previous scores

Still to do:
show/hide roll button. 'show' roll dice button if player selected tile total === roll total TRUE
change messages to the player
show roll total message
-add sounds to the game
-window to select number of players. at the moment it is coded in the html for 2 players

### CODE REVIEW

**Comment by Grub2801** [comment]
Game working

**Comment by Grub2801** [comment]
Not yet linked intro page player or tile select options.

**Comment by Grub2801** [comment]
Bit of a bug with 9 and 12 play options. 
I can only play a 12 tile game once. thereafter the board reverts to a 9 tile option, even if 12 tile is selected on intro page.
Also unsure how to link in the 1 or 2 player option in the intro page. I have commented out some code in the game.js - but not sure how to proceed.

**Comment by Grub2801** [comment]
game working for presentation

**Comment by Grub2801** [comment]
color changes to css

File: game.js [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
-var Game = function(){
+var Game = function() {
+  this.maxTile = 0;
   this.dice = null;
-  this.tilesOpenTotal = 0;
   this.tilesPlayerSelected = [];
-  this.tempPlayerTotal = 0;
+  this.playerTileTotal = 0;
+  this.currentPlayer = "Player1";
+  this.players = {
```
Comment (f3r) [inline game.js:9]: I would have done it like:

``` js
this.player = [{points: 0}, {points: 0}];
```

So I can use later game.player[0].points, it's gramatic sugar, but reads better when i use the class i think

File: game.js [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
 };
 
-Game.prototype.reCalculatePlayerTotal = function(){
-  this.tempPlayerTotal = 0;
-  for (var i = 0; i < this.tempPlayerSelected.length; i++) {
-    this.tempPlayerTotal += this.tempPlayerSelected[i];
+// check player tile selection equals dice roll total
+Game.prototype.calculatePlayerTileSelection = function() {
+  this.playerTileTotal = 0;
+  for ( var i = 0; i < this.tilesPlayerSelected.length; i++ ) {
```
Comment (f3r) [inline game.js:31]:
``` js
this.tilesPlayerSelected.forEach(function(tile) {
  this.playerTileTotal += tile
}
```

Or whatever `tile` actually represents... sorry a bit lost with so many PRs

**Pull request merged**